delivery_recipe/the_coach.md

```diff
@@ -0,0 +1,48 @@
+# The Coach
+
+When delivering a digital service, the Product Owner role is key to success - but is a difficult role to carry out well. The person selected for this role is expected to deliver what’s needed by the business, focus on the users and also play a vital role in protecting and directing the delivery team. It’s not for the faint hearted. They need help to succeed.
+
+The scrum master is a coach for the product owner, and the delivery team. That coaching role is there to support the Product Owner to help them ask the right questions, to help them see the immediate priorities and to help them set a longer term roadmap. That roadmap needs to fit with the availability of the team. There must be the budget to support those and the plan needs to fit with operational expectations too. It’s difficult for the Product Owner to keep on top of those while making decisions for the team each day. That’s where the role of the coach fits in.
```

Comment:
I'd suggest we talk about our 'Delivery Manager' role here rather than scrummaster, and say that a part of the role is what is referred to in scrum as being the 'scrum master'